src/main/java/io/github/gradlenexus/publishplugin/ClosureBackedAction.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2021 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.gradlenexus.publishplugin;
+
+import groovy.lang.Closure;
+import org.codehaus.groovy.runtime.metaclass.MissingMethodExecutionFailed;
+import org.gradle.api.Action;
+import org.gradle.api.InvalidActionClosureException;
+import org.gradle.util.Configurable;
+import java.util.Objects;
+
+/**
+ * Exact copy of [org.gradle.util.internal.ClosureBackedAction] from Gradle 7.6, except.
+
```

Comment:
Extra line, maybe it breaks the javadoc generation on CI...

Reply:
Just pushed a fix.